ui/app/routes/vault/cluster/clients/index.js

```diff
@@ -15,18 +15,27 @@ export default Route.extend(ClusterRoute, {
     },
   },
 
-  async getLicense() {
+  async getActivity(start_time) {
     try {
-      return await this.store.queryRecord('license', {});
+      return await this.store.queryRecord('clients/activity', { start_time });
     } catch (e) {
       // ARG TODO handle
       return e;
     }
   },
 
-  async getActivity(start_time) {
+  async getVersionHistory() {
     try {
-      return await this.store.queryRecord('clients/activity', { start_time });
+      return await this.store.queryRecord('clients/version-history', {});
+    } catch (e) {
+      // ARG TODO handle
+      return e;
```

Comment:
Failing this call should allow the UI to fallback gracefully so I'd go ahead and return null rather than the error. Would likely be a permissions issue anyway